common/autoware_auto_geometry/package.xml

```diff
@@ -4,7 +4,7 @@
   <name>autoware_auto_geometry</name>
   <version>1.0.0</version>
   <description>Geometry related algorithms</description>
-  <maintainer email="opensource@apex.ai">Apex.AI, Inc.</maintainer>
+  <maintainer email="satoshi.ota@tier4.jp">Satoshi Ota</maintainer>
   <license>Apache License 2.0</license>
 
   <buildtool_depend>ament_cmake_auto</buildtool_depend>
```

Comment:
Let's keep them as the author to respect the original developer!

```suggestion
    <author email="opensource@apex.ai">Apex.AI, Inc.</author>

    <buildtool_depend>ament_cmake_auto</buildtool_depend>
```

Reply:
OK! Thanksf for your suggestion.